### PR TITLE
test: add test for ZSet.ZKeys

### PR DESCRIPTION
**Pull request by her-cat**
Related issue: #503 

I completed this unit test by referring to [TestTx_SKeys](https://github.com/nutsdb/nutsdb/blob/f7c4a82b75c2f437feccd9499ca1242a169cd858/tx_set_test.go#L401).

### CODE REVIEW

**Comment by bigboss2063** [comment]
Hi, there seems to be some problems with this method `ZKeys` itself. I'm sorry to notice it so late. Do you want to try another uint test like `ZRangeByScore`?

**Comment by her-cat** [comment]
Sure, I can write the unit test for 'ZRangeByScore'. Write in the new PR?

**Comment by bigboss2063** [comment]
> Sure, I can write the unit test for 'ZRangeByScore'. Write in the new PR?

Yes, thanks！

File: tx_zset_test.go [hunk 1]
```diff
@@ -334,6 +335,34 @@ func TestTx_ZRank(t *testing.T) {
...
+		num := 3
+		for i := 0; i < num; i++ {
+			txZAdd(t, db, bucket, []byte(fmt.Sprintf(key, i)), val, float64(i), nil, nil)
+		}
+
+		txZKeys(t, db, bucket, "*", func(key string) bool {
+			return true
+		}, num, nil)
+
+		var keys []string
```
[the remark this answers is not in the document]
Comment (her-cat) [reply tx_zset_test.go:354]: 好的，我会一起处理这些问题。

**Comment by her-cat** [comment]
@damotiansheng please take a look.

**Comment by bigboss2063** [comment]
Thanks to your contribution!

**Pull request merged**